src/System.CommandLine/CommandLineConfiguration.cs

```diff
@@ -11,12 +11,28 @@
 
 namespace System.CommandLine
 {
+    /// <summary>
+    /// Represents the configuration used by the <see cref="Parser"/>.
+    /// </summary>
     public class CommandLineConfiguration
     {
         private readonly IReadOnlyCollection<InvocationMiddleware> _middlewarePipeline;
         private readonly Func<BindingContext, IHelpBuilder> _helpBuilderFactory;
         private readonly SymbolSet _symbols = new SymbolSet();
 
+        /// <summary>
+        /// Initializes a new instance of the CommandLineConfiguration class.
+        /// </summary>
+        /// <param name="symbols">The symbols to parse.</param>
+        /// <param name="argumentDelimiters">The characters used to delimit arguments.</param>
```

Comment:
It might be helpful to indicate the default delimiters.

Reply:
OK included this, take a look.